src/index.js

```diff
@@ -134,11 +136,9 @@ module.exports = {
       if (error) {
         throw error;
       } else {
-        const { summary, errors } = formatResults({ results, thresholds });
+        const { summary, message, errors } = formatResults({ results, thresholds });
         console.log(summary);
-        show({
-          summary: JSON.stringify(summary, null, 2),
-        });
+        show(message);
```

Comment:
I think you need to pass an object to `show`:
https://docs.netlify.com/configure-builds/build-plugins/create-plugins/#logging